mtags/src/main/scala-2/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -50,6 +50,7 @@ import org.eclipse.lsp4j.TextEdit
 
 case class ScalaPresentationCompiler(
     buildTargetIdentifier: String = "",
+    buildTargetName: Option[String] = None,
```

Comment:
Potentially, we could somehow use `buildTargetIdentifier` instead

Reply:
But that's an URI, so it's lengthy and awkward to include in a file name.

Reply:
Do you mean to extract a "meaningful part" (a name)? That could be problematic, I think it's very build server specific and not regulated in any way. Or do you mean to add the full URI to the report but not in the file name? That we can do, but then I prefer the version where the path to the file was in the report. Same thing but more info.

Reply:
Let's maybe leave it at the name then, sorry for the confusion